### PR TITLE
add pcre/8.41

### PR DESCRIPTION
**Pull request by ericLemanissier**
pcre/8.41

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/pcre/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
```
Comment (Croydon) [inline recipes/pcre/all/conanfile.py]: please remove all three of this meta lines

File: recipes/pcre/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+
+class PCREConan(ConanFile):
+    name = "pcre"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.pcre.org"
+    author = "Bincrafters <bincrafters@gmail.com>"
+    description = "Perl Compatible Regular Expressions"
+    license = "BSD"
+    exports = ["LICENSE.md"]
```
Comment (uilianries) [inline recipes/pcre/all/conanfile.py]:
```suggestion
```

File: recipes/pcre/all/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import os
+
+
+class PCREConan(ConanFile):
+    name = "pcre"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.pcre.org"
+    author = "Bincrafters <bincrafters@gmail.com>"
+    description = "Perl Compatible Regular Expressions"
+    license = "BSD"
```
Comment (uilianries) [inline recipes/pcre/all/conanfile.py]:
```suggestion
    license = "BSD-3-Clause"
```
I've read the pcre's license file. There is a mention about BSD, but reading all license file you can file all clauses from version 3 there.

File: recipes/pcre/all/conanfile.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.remove("fPIC")
+
+    def configure(self):
+        if not self.options.build_pcrecpp:
+            del self.settings.compiler.libcxx
```
Comment (uilianries) [inline recipes/pcre/all/conanfile.py]:
```suggestion
            del self.settings.compiler.libcxx
            del self.settings.compiler.cppstd
```

**Pull request merged**